lib/relation-definition.js

```diff
@@ -1338,7 +1338,8 @@ RelationDefinition.belongsTo = function(modelFrom, modelToRef, params) {
     get: function() {
       var relation = new BelongsTo(definition, this);
       var relationMethod = relation.related.bind(relation);
-      relationMethod.getAsync = relation.getAsync.bind(relation);
+      relationMethod.get = relation.get.bind(relation);
+      relationMethod.getAsync = relationMethod.get; // deprecated alias
```

Comment:
Taking care of all this now, but I'll adhere to ES5 style JS, as in the rest of the file.